css-highlight-api-1/Overview.bs

```diff
@@ -309,12 +309,12 @@ Painting</h4>
 					<body>Lorem Ipsum.
 					<script>
 						let r1 = new Range();
```

Comment:
For clarity, I would recommend making a text node variable and using that in the calls to setStart/setEnd.